enricher/rhcc/rhcc.go

```diff
@@ -0,0 +1,45 @@
+package rhcc
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/claircore/rhel/rhcc"
+)
+
+type Enricher struct{}
+
+var (
+	_ driver.Enricher = (*Enricher)(nil)
+)
+
+const (
+	// Type is the type of data returned from the Enricher's Enrich method.
+	Type = `message/vnd.clair.map.layer; enricher=clair.rhcc`
+)
+
+func (e *Enricher) Name() string { return "rhcc" }
+
+func (e *Enricher) Enrich(ctx context.Context, g driver.EnrichmentGetter, r *claircore.VulnerabilityReport) (string, []json.RawMessage, error) {
+	problematicPkgs := make(map[string]string)
```

Comment:
Shouldn't this just be a `map[LayerID]bool`? It just needs to be a list of layers, right? I don't think `map[PkgID]LayerID` buys us a lot.

Reply:
My thought was that it might be useful in the future for people to be able to cross-reference _which_ rhcc layer it is too and the added data seems negligible size-wise and nil complexity-wise.

Reply:
I guess the `rhcc` packages should be at most one per layer. OK.